processing file asmFiles/branch_unit.asm (1 of 15)
processing file asmFiles/count_days.asm (2 of 15)
processing file asmFiles/empty.asm (3 of 15)
processing file asmFiles/fib.asm (4 of 15)
processing file asmFiles/halt_unit.asm (5 of 15)
processing file asmFiles/jr_test.asm (6 of 15)
processing file asmFiles/jump_unit.asm (7 of 15)
processing file asmFiles/mergesort.asm (8 of 15)
processing file asmFiles/mult.asm (9 of 15)
processing file asmFiles/mult_procedure.asm (10 of 15)
processing file asmFiles/quick.asm (11 of 15)
--- memsim.hex	2020-10-25 13:43:47.348772454 -0400
+++ memcpu.hex	2020-10-25 13:43:47.286770224 -0400
@@ -173,67 +173,71 @@
 :0400E40027BD0050E4
 :0400E50003E000082C
 :0400E700FFFFFFFF19
+:040F560000003D60FA
+:040F5700000002E0B4
+:040F580000003FBC9A
+:040F59000000000292
+:040F5A000000000192
+:040F660000003DA0AA
+:040F6700000002E0A4
+:040F680000003FBC8A
+:040F69000000000381
+:040F6A000000000281
+:040F760000003DE05A
+:040F7700000002E094
+:040F780000003FBC7A
+:040F79000000000470
+:040F7A000000000370
+:040F860000003E2009
+:040F8700000002E084
+:040F880000003FBC6A
+:040F8900000000055F
+:040F8A00000000045F
 :040F960000003E60B9
 :040F9700000002E074
 :040F980000003FBC5A
-:040F9900000000084C
-:040F9A00000000074C
-:040FA0000000000746
-:040FA1000000003F0D
-:040FA2000000000744
-:040FA3000000003F0B
-:040FA4000000000742
-:040FA5000000000741
+:040F9900000000064E
+:040F9A00000000054E
 :040FA60000003EA069
 :040FA700000002E064
 :040FA80000003FBC4A
-:040FA900000000063E
-:040FAA00000000073C
+:040FA900000000073D
+:040FAA00000000063D
 :040FB60000003EE019
 :040FB700000002E054
 :040FB80000003FBC3A
-:040FB900000000092B
-:040FBA00000000092A
-:040FC0000000000924
-:040FC10000000054D8
-:040FC2000000000823
-:040FC3000000004BDF
-:040FC4000000000920
-:040FC5000000000820
+:040FB900000000082C
+:040FBA00000000072C
 :040FC60000003F20C8
 :040FC700000002E044
 :040FC80000003FBC2A
-:040FC9000000000321
-:040FCA00000000091A
-:040FD0000000000914
-:040FD1000000004BD1
-:040FD2000000000219
-:040FD3000000001109
-:040FD4000000000910
-:040FD5000000000216
+:040FC900000000091B
+:040FCA00000000081B
 :040FD60000003F6078
 :040FD700000002E034
 :040FD80000003FBC1A
-:040FD9000000000113
+:040FD9000000000A0A
 :040FDA00000000090A
 :040FE0000000000904
-:040FE10000000011FB
-:040FE3000000000901
+:040FE1000000004BC1
+:040FE2000000000902
+:040FE3000000004BBF
 :040FE4000000000900
+:040FE50000000009FF
 :040FE60000003FA028
 :040FE700000003788B
 :040FE80000003FBC0A
 :040FEA0000000009FA
-:040FEF0000000009F5
-:040FF0000000000DF0
-:040FF10000000011EB
+:040FEF0000000011ED
+:040FF0000000003FBE
+:040FF1000000004BB1
 :040FF2000000001BE0
 :040FF30000000021D9
-:040FF40000000029D0
+:040FF4000000000DEC
 :040FF50000000034C4
-:040FF6000000003FB8
-:040FF7000000004BAB
-:040FF80000000054A1
+:040FF60000000029CE
+:040FF70000000054A2
+:040FF80000000009EC
 :040FFA0000003FF0C4
 :040FFB000000000CE6
 :00000001FF
processing file asmFiles/sw_check.asm (12 of 15)
processing file asmFiles/testfile.asm (13 of 15)
processing file asmFiles/test.loadstore.asm (14 of 15)
processing file asmFiles/test.rtype.asm (15 of 15)
processing file asmFiles/quick.asm (1 of 1)
--- memsim.hex	2020-10-25 13:44:51.608084972 -0400
+++ memcpu.hex	2020-10-25 13:44:51.532082237 -0400
@@ -159,31 +159,43 @@
 :04009E0027BD00403A
 :04009F0003E0000872
 :0400A000FFFFFFFF60
-:0401BA00000000063B
+:0401A6000000000352
+:0401A800000006A8A5
+:0401A900000001EC65
+:0401AA00000007D872
+:0401AB000000000749
+:0401AC000000000748
+:0401B0000000000645
+:0401B200000006D073
+:0401B300000001EC5B
+:0401B400000007D868
+:0401B5000000000640
+:0401B600000000073E
+:0401BA00000000053C
 :0401BC00000006F841
 :0401BD00000001EC51
 :0401BE00000007D85E
-:0401BF000000000735
-:0401C0000000000635
-:0401C4000000000631
+:0401BF000000000537
+:0401C0000000000734
+:0401C4000000000433
 :0401C600000007200E
 :0401C700000001EC47
 :0401C800000007D854
-:0401C900000000082A
+:0401C900000000042E
 :0401CA00000000072A
-:0401CE000000000726
+:0401CE00000000032A
 :0401D00000000748DC
 :0401D100000001EC3D
 :0401D200000007D84A
-:0401D3000000000523
+:0401D3000000000325
 :0401D4000000000720
-:0401D800000000041F
+:0401D8000000000221
 :0401DA0000000770AA
 :0401DB00000001EC33
 :0401DC00000007D840
-:0401DD00000000031B
+:0401DD00000000021C
 :0401DE000000000716
-:0401E2000000000217
+:0401E2000000000118
 :0401E4000000079878
 :0401E500000001EC29
 :0401E600000007D836
@@ -194,13 +206,13 @@
 :0401F000000007D82C
 :0401F2000000000702
 :0401F6000000000104
-:0401F7000000000202
-:0401F8000000000300
-:0401F90000000004FE
-:0401FA0000000005FC
-:0401FB0000000005FB
-:0401FC0000000006F9
-:0401FD0000000008F6
+:0401F7000000000301
+:0401F80000000004FF
+:0401F90000000005FD
+:0401FA0000000006FB
+:0401FB0000000008F8
+:0401FC0000000005FA
+:0401FD0000000002FC
 :0401FE0000000800F5
 :0401FF000000000CF0
 :00000001FF
processing file asmFiles/quick.asm (1 of 1)
--- memsim.hex	2020-10-25 13:45:19.719096610 -0400
+++ memcpu.hex	2020-10-25 13:45:19.650094127 -0400
@@ -159,31 +159,43 @@
 :04009E0027BD00403A
 :04009F0003E0000872
 :0400A000FFFFFFFF60
-:0401BA00000000063B
+:0401A6000000000352
+:0401A800000006A8A5
+:0401A900000001EC65
+:0401AA00000007D872
+:0401AB000000000749
+:0401AC000000000748
+:0401B0000000000645
+:0401B200000006D073
+:0401B300000001EC5B
+:0401B400000007D868
+:0401B5000000000640
+:0401B600000000073E
+:0401BA00000000053C
 :0401BC00000006F841
 :0401BD00000001EC51
 :0401BE00000007D85E
-:0401BF000000000735
-:0401C0000000000635
-:0401C4000000000631
+:0401BF000000000537
+:0401C0000000000734
+:0401C4000000000433
 :0401C600000007200E
 :0401C700000001EC47
 :0401C800000007D854
-:0401C900000000082A
+:0401C900000000042E
 :0401CA00000000072A
-:0401CE000000000726
+:0401CE00000000032A
 :0401D00000000748DC
 :0401D100000001EC3D
 :0401D200000007D84A
-:0401D3000000000523
+:0401D3000000000325
 :0401D4000000000720
-:0401D800000000041F
+:0401D8000000000221
 :0401DA0000000770AA
 :0401DB00000001EC33
 :0401DC00000007D840
-:0401DD00000000031B
+:0401DD00000000021C
 :0401DE000000000716
-:0401E2000000000217
+:0401E2000000000118
 :0401E4000000079878
 :0401E500000001EC29
 :0401E600000007D836
@@ -194,13 +206,13 @@
 :0401F000000007D82C
 :0401F2000000000702
 :0401F6000000000104
-:0401F7000000000202
-:0401F8000000000300
-:0401F90000000004FE
-:0401FA0000000005FC
-:0401FB0000000005FB
-:0401FC0000000006F9
-:0401FD0000000008F6
+:0401F7000000000301
+:0401F80000000004FF
+:0401F90000000005FD
+:0401FA0000000006FB
+:0401FB0000000008F8
+:0401FC0000000005FA
+:0401FD0000000002FC
 :0401FE0000000800F5
 :0401FF000000000CF0
 :00000001FF
processing file asmFiles/quick.asm (1 of 1)
--- memsim.hex	2020-10-25 13:46:18.930227457 -0400
+++ memcpu.hex	2020-10-25 13:46:18.831223895 -0400
@@ -173,67 +173,71 @@
 :0400E40027BD0050E4
 :0400E50003E000082C
 :0400E700FFFFFFFF19
+:040F560000003D60FA
+:040F5700000002E0B4
+:040F580000003FBC9A
+:040F59000000000292
+:040F5A000000000192
+:040F660000003DA0AA
+:040F6700000002E0A4
+:040F680000003FBC8A
+:040F69000000000381
+:040F6A000000000281
+:040F760000003DE05A
+:040F7700000002E094
+:040F780000003FBC7A
+:040F79000000000470
+:040F7A000000000370
+:040F860000003E2009
+:040F8700000002E084
+:040F880000003FBC6A
+:040F8900000000055F
+:040F8A00000000045F
 :040F960000003E60B9
 :040F9700000002E074
 :040F980000003FBC5A
-:040F9900000000084C
-:040F9A00000000074C
-:040FA0000000000746
-:040FA1000000003F0D
-:040FA2000000000744
-:040FA3000000003F0B
-:040FA4000000000742
-:040FA5000000000741
+:040F9900000000064E
+:040F9A00000000054E
 :040FA60000003EA069
 :040FA700000002E064
 :040FA80000003FBC4A
-:040FA900000000063E
-:040FAA00000000073C
+:040FA900000000073D
+:040FAA00000000063D
 :040FB60000003EE019
 :040FB700000002E054
 :040FB80000003FBC3A
-:040FB900000000092B
-:040FBA00000000092A
-:040FC0000000000924
-:040FC10000000054D8
-:040FC2000000000823
-:040FC3000000004BDF
-:040FC4000000000920
-:040FC5000000000820
+:040FB900000000082C
+:040FBA00000000072C
 :040FC60000003F20C8
 :040FC700000002E044
 :040FC80000003FBC2A
-:040FC9000000000321
-:040FCA00000000091A
-:040FD0000000000914
-:040FD1000000004BD1
-:040FD2000000000219
-:040FD3000000001109
-:040FD4000000000910
-:040FD5000000000216
+:040FC900000000091B
+:040FCA00000000081B
 :040FD60000003F6078
 :040FD700000002E034
 :040FD80000003FBC1A
-:040FD9000000000113
+:040FD9000000000A0A
 :040FDA00000000090A
 :040FE0000000000904
-:040FE10000000011FB
-:040FE3000000000901
+:040FE1000000004BC1
+:040FE2000000000902
+:040FE3000000004BBF
 :040FE4000000000900
+:040FE50000000009FF
 :040FE60000003FA028
 :040FE700000003788B
 :040FE80000003FBC0A
 :040FEA0000000009FA
-:040FEF0000000009F5
-:040FF0000000000DF0
-:040FF10000000011EB
+:040FEF0000000011ED
+:040FF0000000003FBE
+:040FF1000000004BB1
 :040FF2000000001BE0
 :040FF30000000021D9
-:040FF40000000029D0
+:040FF4000000000DEC
 :040FF50000000034C4
-:040FF6000000003FB8
-:040FF7000000004BAB
-:040FF80000000054A1
+:040FF60000000029CE
+:040FF70000000054A2
+:040FF80000000009EC
 :040FFA0000003FF0C4
 :040FFB000000000CE6
 :00000001FF
